style.css

```diff
@@ -61,6 +61,33 @@
   padding: 10px 15px 15px 10px;
 }
 
+.mobile-menu {
```

Comment:
- [ ] Kindly make sure that the mobile menu follows the Figma design as closely as possible

|Your page|Figma design|
|-|-|
|![Screenshot_20221222_073352](https://user-images.githubusercontent.com/25227022/209072404-7cb4725a-e5e3-4822-bad2-9df013d464bc.png)|![Screenshot_20221222_073854](https://user-images.githubusercontent.com/25227022/209072597-218e6623-f79e-40e4-b49b-23448b59d6a5.png)|

Reply:
can you please give me a better description on what I can do to make the project better

Reply:
@PrinceBadru, Make sure to style the mobile menu as in the Figma design. the menu should take up all the space on the page when open, and the font should have more weight. check the design above ☝🏼